### DISCUSSION TITLE
Anvil not forking URL

### DISCUSSION THREAD

**Discussion by mustafaosmancoding** [Jeiwan/uniswapv3-book#72] [General]
Hi guy, I am receiving this error:

""" Error:  contract source info format must be `<path>:<contractname>` or `<contractname>`  """

When running:

""" $ forge script scripts/DeployDevelopment.s.sol --broadcast --fork-url http://localhost:8545 --private-key 0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80 """

Do you know what the issue might be?

**Comment by Jeiwan** [comment]
I cannot reproduce this on the `main` branch. This looks like a Foundry issue, or a misconfiguration. Try running `foundryup`.
